lib/jsdom/level1/core.js

```diff
@@ -245,7 +245,7 @@ core.DOMImplementation.prototype = {
 };
 
 core.Node = function Node(ownerDocument) {
-  EventTarget.call(this);
+  EventTarget.init.call(this);
```

Comment:
The constructor of generated classes always does argument conversion and then redirects to generated.init, which does object creation (in cases where non is provided, i.e. instancing of objects without a constructor (HTMLElements, etc.)) and then forwards to impl.init which is the "proper" constructor.
I want to keep the init method as close to a real constructor as possible - which also has the added benefit of being able to just provide a `function Klass` definition as-is as both interface and init.